mico-core/src/main/java/io/github/ust/mico/core/persistence/MicoApplicationRepository.java

```diff
@@ -42,4 +43,7 @@
     @Depth(3)
     Optional<MicoApplication> findByShortNameAndVersion(String shortName, String version);
     
+    @Query("MATCH (a:MicoApplication)-[i:INCLUDES_SERVICE]-(s:MicoService) WHERE s.shortName = {shortName} AND s.version = {version} RETURN COLLECT(a) AS applications")
+    List<MicoApplication> findAllUsedByService(@Param("shortName") String shortName, @Param("version") String version);
```

Comment:
Misleading name of method. Proposal: `findAllThatUsesSpecificService` or similar

Reply:
Renaming to **findAllByUsedService**.